src/NHibernate/Cfg/Loquacious/IDbIntegrationConfigurationProperties.cs

```diff
@@ -25,6 +25,7 @@ public interface IDbIntegrationConfigurationProperties
 		void Batcher<TBatcher>() where TBatcher : IBatcherFactory;
 		short BatchSize { set; }
 		bool OrderInserts { set; }
+		bool TrackSessionId { set; }
```

Comment:
I would more see that among session factory settings, on IFluentSessionFactoryConfiguration, as a method taking a boolean.

I have to say I never updated the loquacious cfg when adding parameters, and it is likely there are a bunch of them lacking.

Moreover looking at it, it seems that we should reflect changes done on the extension based `IDbIntegrationConfigurationProperties` to the fluent `IDbIntegrationConfiguration` too.

Since it is a public interface by the way, this is technically a breaking change. Should we do it for a revision fix?

Reply:
I removed the member from `IDbIntegrationConfigurationProperties` in order to avoid a breaking change. IMO , the loquacious cfg should be updated in a separate PR, where the missing parameters would be added.